src/garage/experiment/local_tf_runner.py

```diff
@@ -187,7 +187,7 @@ def save(self, epoch, paths=None):
 
         Args:
             itr(int): Index of iteration (epoch).
-            paths(dict): Batch of samples after preprocessed.
+            paths(None or dict): Batch of samples after preprocessed.
```

Comment:
you can say `paths(dict)` and says that if paths is none, no paths will be logged to the snapshot.